src/Utilities/ToolLocationHelper.cs

```diff
@@ -61,6 +61,12 @@ public enum TargetDotNetFrameworkVersion
         Version451 = 6,
 
         /// <summary>
+        /// version 4.5.2. Enum is out of order because it was shipped out of band from a Visual Studio update
+        /// without a corresponding SDK release.
+        /// </summary>
+        Version452 = 9,
```

Comment:
I originally put it in order, but then changed to order by version. Neither seemed very good.